Source/Core/VideoCommon/AbstractTexture.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+#include "Common/MathUtil.h"
+#include "VideoCommon/TextureConfig.h"
+#include "VideoCommon/VideoCommon.h"
+
```

Comment:
Would you mind if we saved this for another PR?  That would touch a bunch more files and I would like to avoid bloating this code review more than it is..